CHANGELOG.md

```diff
@@ -14,11 +14,17 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
     * #### Changed
+        * Option string can now be a python dictionary instead of a string. It will be converted as needed (Fix #661)
```

Comment:
* Use markdown to link to #661 (we should make that standard practice)
* We should move detailed documentation to the appropriate places (RST / docstring)